plugins/Denizen/scripts/restraining_order.dsc

```diff
@@ -0,0 +1,33 @@
+restraining_order:
+    type: item
+    display name: <&f>Restraining Order
+    material: paper
+    flags:
+        uuid: <util.random_uuid>
+    lore:
+        - <&7>Right click to assign to a player!
+
+restraining_order_handler:
+    type: world
+    debug: true
+    events:
+        on player right clicks entity with:restraining_order type:player:
+            - if <context.item.has_flag[restrained]>:
+                - stop
+            - define slot <player.held_item_slot>
+            - inventory flag slot:<[slot]> restrained:<context.entity.if_null[<empty>]>
+            - define item <player.item_in_hand>
+            - inventory adjust slot:<[slot]> lore:<&7>Attached<&sp>to<&sp><[item].flag[restrained].name>
+            - flag <context.entity> restraining_order_<player>:<player>
+            - flag <context.entity> restraining_order<player>
+            - narrate "<&7>You have <[item].flag[restrained].name> set for your restraining order!"
```

Comment:
This sounds a bit strange....?
_This is just my opinion so you don't have to worry about this comment_ but I think it might sound a bit nicer if it was something like
`You now have a restraining order against PLAYER!`
Again, it's just my opinion you can keep it if you think your version sounds better :)